content/api/dashboard_lists/dashboard_lists_get_all.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Get all Dashboard Lists
+type: apicontent
+order: 8.2
+external_redirect: /api/#get-all-dashboard-lists
+---
+
+## Get all Dashboard Lists
+
+Fetch all of your existing dashboard list’s definitions.
```

Comment:
I'm not sure what the Datadog standard is for plural possessive. I think this probably should be either `lists' definitions` or `lists's definitions`. Probably worth tagging in @johnaxel to comment.

Reply:
lists's is always wrong. but lists' definitions is ok. but wait. is this getting all the lists? so dashboard list definitions is better.

Reply:
I agree that "dashboard list definitions" is best.